data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -74,6 +74,25 @@
 ]
 TILES_ROUND_NUM_DECIMALS = 2
 
+# Controlling Tile user experience columns
+THRESHOLD_COUNT_TO_SHOW_FIELD_NAME = "Thresholds"
+TILES_ISLAND_AREAS_THRESHOLD_COUNT = 4
+TILES_PUERTO_RICO_THRESHOLD_COUNT = 5
+TILES_NATION_THRESHOLD_COUNT = 21
+
+# Note that the FIPS code is a string
+# The FIPS codes listed are:
+# 60: American Samoa, 66: Guam, 69: N. Mariana Islands, 78: US Virgin Islands
+TILES_ISLAND_AREA_FIPS_CODES = ["60", "66", "69", "78"]
+TILES_PUERTO_RICO_FIPS_CODE = ["72"]
+
+# Constant to reflect UI Experience version
```

Comment:
nit: maybe a slightly more elaborate comment so that a developer unfamiliar with the codebase can understand? E.g., the comment you have, but also something like "This variable will be used by the front-end to show a different side panel. The different side panels are necessary because the nation, Puerto Rico, and island areas all have different fields to display, respectively." 

Something like that.